index.rb

```diff
@@ -0,0 +1,30 @@
+require_relative './module'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(*list)
+    @list = list
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
+end
+
+list = MyList.new(1, 2, 3, 4)
+# Test #all?
+puts(list.all? { |e| e < 5 })
+# => true
+puts(list.all? { |e| e > 5 })
+# => false
+
+# Test #any?
+puts(list.any? { |e| e == 2 })
+# => true
+puts(list.any? { |e| e == 5 })
+# => false
+
+# Test #filter
+puts list.filter(&:even?)
+# => [2, 4]
```

Comment:
- [ ] As you mentioned here and as mentioned in the project requirements this should return `[2, 4]` but for you, the result is a little bit different, please kindly fix that 🙏🏾

<img width="739" alt="Screenshot 2023-06-16 at 13 39 33" src="https://github.com/seyioyemade/my-enumerable/assets/74501523/fd322b81-0a91-4a0f-b650-62966ff8573a">